main.tf

```diff
@@ -226,7 +226,7 @@ resource "aws_ec2_client_vpn_network_association" "default" {
   subnet_id              = var.associated_subnets[count.index]
 
   security_groups = concat(
-    [module.vpn_security_group.id],
+    compact([module.vpn_security_group.id]),
     local.associated_security_group_ids
   )
```

Comment:
Hmm maybe we should run compact over the whole thing

Reply:
cc: @nyf-nyf one last comment

Reply:
@nitrocode Done